### ISSUE TITLE
Output blockchain APIs in a format that can be used for documentation

### ISSUE THREAD

**Issue by kauffj** [lbryio/lbrycrd#131]
lbry.tech will be used to showcase the API signatures for the daemon, blockchain, and any other key components.

**Comment by kauffj** [comment]
@lyoshenka did this get discussed on blockchain weekly?

**Comment by BrannonKing** [comment]
Waiting on #44

**Comment by BrannonKing** [comment]
Is this a duplicate of #143 ?

**Comment by lyoshenka** [comment]
yes. ill close the other one.

**Comment by BrannonKing** [comment]
Also, see https://github.com/lbryio/lbry.tech/issues/42
and https://github.com/lbryio/lbry/blob/master/scripts/generate_json_api.py

I'm starting on this now and planning to pursue the jrgen plan.

**Comment by rynomad** [comment]
@BrannonKing re: jrgen... if you'd like, take a look at [this script](https://github.com/rynomad/lbry-js/blob/master/scripts/jrgen_crd.js) I wrote for lbry-js; it's pretty barebones but generates a jrgen compatible schema file dynamically from the existing lbrycrd 'help' command.

**Comment by BrannonKing** [comment]
@rynomad , thanks for posting that script. Unfortunately, my Python script was generally completed before I got this message yesterday. Hence, I think I'll proceed with my script. 

I was unable to run your script. I'm not sure I followed your instructions correctly. `npm --version` reports 3.5.2. Do I need some other version? (There was no `setup` command on mine.)
